airflow/models/dag.py

```diff
@@ -1894,6 +1895,13 @@ def set_is_paused(self,
         try:
             for dag_model in dag_models:
                 dag_model.is_paused = is_paused
+            to_email_address = self.default_args['email']
+            email_subject = 'Airflow DAG {} Toggled On/Off'.format(self.dag_id)
```

Comment:
Can we please be specific I/E send different content when its toggler ON vs toggled OFF based on is_paused instead of being generic here? Also some unit tests